Documentation/subcommands/export.md

```diff
@@ -0,0 +1,17 @@
+# rkt export
+
+Exports an exited, single-app pod to an App Container Image (.aci)
+
+```
+$ rkt export UUID .aci
+```
+
+## Options
+
+| Flag | Default | Options | Description |
+| --- | --- | --- | --- |
+| `--overwrite` |  `false` | `true` or `false` | Overwrite the output aci if it exists  |
```

Comment:
@robszumski I've always only used a comma with an 'If' conditional phrase if it's at the beginning of the sentence. See https://www.ego4u.com/en/cram-up/writing/comma?10